### PR TITLE
Change the JS object name with a field [Delivers #85500346]

### PR DESCRIPTION
**Pull request by Petervw**
Fixes https://wordpress.org/support/topic/note-change-of-global-function-ga-_gatracker?replies=8

### CODE REVIEW

File: frontend/views/tracking-universal.php [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
-	})(window,document,'script','//www.google-analytics.com/analytics.js','__gaTracker');
+	})(window,document,'script','//www.google-analytics.com/analytics.js',<?php echo $object_name; ?>);
 
 <?php
 // List the GA elements from the class-ga-js.php
 if ( count( $gaq_push ) >= 1 ) {
 	foreach ( $gaq_push as $item ) {
 		if ( ! is_array( $item ) ) {
-			echo '	__gaTracker('.$item.");\n";
+			echo '	' . $object_name . '('.$item.");\n";
```
Comment (andizer) [inline frontend/views/tracking-universal.php]: code style?

File: frontend/class-universal.php [hunk 2]
```diff
@@ -134,6 +141,8 @@ public function tracking( $return_array = false ) {
 
 			$ga_settings = $this->options; // Assign the settings to the javascript include view
 
+			$object_name = $this->get_tracker_object_name();
```
Comment (andizer) [inline frontend/class-universal.php]: You aren't doing anything within this method with this variable?

Comment (andizer) [inline frontend/class-universal.php]: Can't you declare this in the else on line 151? Because it's only used in tracking-universal.php view

**Comment by Petervw** [comment]
Update: Merged trunk into this PR. The settings API here available with the new object setting for universal.

**Pull request closed without merging**